src/App.js

```diff
@@ -34,10 +32,20 @@ const tweetsArray = [
   },
 ];
 
+const Test = ({ tweets }) => (
+  <>
+    {tweets.map((tweet, index) => (
+      <div key={index}>
```

Comment:
why the extra div?
and while index makes the error go away, a nicer way would be to use an actual id.